maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -518,10 +520,9 @@ else if ( !"auto".equals( styleColor ) )
             throw new IllegalArgumentException( "Invalid color configuration option [" + styleColor
                 + "]. Supported values are (auto|always|never)." );
         }
-        else if ( cliRequest.commandLine.hasOption( CLIManager.BATCH_MODE )
-            || cliRequest.commandLine.hasOption( CLIManager.LOG_FILE ) )
+        else
```

Comment:
this else-statement can now be removed, because the color mode has already been applied